modules/price/src/Element/Price.php

```diff
@@ -61,9 +61,9 @@ public function getInfo() {
    * {@inheritdoc}
    */
   public static function valueCallback(&$element, $input, FormStateInterface $form_state) {
-    if ($input !== FALSE && $input !== NULL) {
+    if ($input !== FALSE && $input !== NULL && !empty($input['amount']) && !empty($input['currency_code'])) {
```

Comment:
This prevents null values from being passed in. We should fix this in 8.x-2.x HEAD right?